cloudmarker/clouds/gcpcloud.py

```diff
@@ -6,10 +6,13 @@
 
 
 import json
+import logging
 
 from google.oauth2 import service_account
 from googleapiclient import discovery
 
+_log = logging.getlogger(__name__)
```

Comment:
Does this code work for you? This line should lead to the following error:

```
AttributeError: module 'logging' has no attribute 'getlogger'
```

The actual function name is `getLogger`.

Reply:
I did not run the code after making changes :P